src/pages/Filing/ViewUserProfile/UserInformation.tsx

```diff
@@ -9,6 +9,7 @@ export default function UserInformation({
 }: {
   data: UserProfileObject;
 }): JSX.Element {
+  console.log(data);
```

Comment:
Could you kill this console.log then this is good to go. 👍

Reply:
Gone